README.md

```diff
@@ -1,29 +1,29 @@
 # MediathekViewWeb -> [mediathekviewweb.de](https://mediathekviewweb.de/)
 
-MediathekViewWeb ist eine simple, alternative Oberfläche für die Filmlisten des [MediathekView Projekts](https://mediathekview.de/).
+MediathekViewWeb ist eine einfache Browser-Oberfläche für den Zugriff auf die Filmliste des [MediathekView Projekts](https://mediathekview.de/).
 
-Im Gegensatz zu MediathekView muss man hier jedoch keine Filmliste herunterladen, die Suche steht also sofort zur Verfügung.
-Da die Abfragen auf dem Server durchgeführt werden, sind die Anforderungen an das Endgerät minimal - in Kombination mit dem Wegfall von Java lässt sich die Website auch auf bspw. Smartphones und Tablets nutzen. 
+Im Gegensatz zu MediathekView muss bei MediatheViewWeb weder ein Programm installiert noch eine Filmliste heruntergeladen werden: die Suche steht deshalb augenblicklich im Browser zur Verfügung.
+Da die Abfragen auf dem Server durchgeführt werden, sind die Anforderungen an das Endgerät (Browser) minimal - da kein Java installiert werden muss, lässt sich die Website auch auf Smartphones und Tablets nutzen. 
```

Comment:
The "Endgerät" isn't the browser.
The combination of both, low requirements and no java makes it suitable for mobile devices, your text suggests its just because of java.